Makefile

```diff
@@ -12,3 +12,6 @@ push: ## Push image
 
 pull: ## Pull image 
 	DOCKER_BUILDKIT=1 docker-compose -f docker-compose.yml pull
+
+doxy-doc: ## Generate Doxygen documentation
+	docker run --rm -it -v $(PWD):/stm32-app $(shell docker build -q -f Doxygen/Dockerfile .) /bin/sh -c "doxygen Doxygen/config/Doxyfile"
```

Comment:
I think adding quotes to the path fixes the error I got when the folder has spaces. 
```"$(PWD):/stm32-app"```. 

I tried using that and the command runs without errors.